README.md

```diff
@@ -12,394 +12,61 @@
-
-This modality provides three options:
-
-1. `DerivedFromCollectionName`: This modality will signal the contract to attempt to install a new version 1.1.1 instance of the CEP-78 contract. The contract package hash and the access URef will be saved in the installing account's `NamedKeys` as `cep78_contract_package_<collection_name>` and `cep78_contract_package_access_<collection_name>`.
-2. `V_1_0_standard`: This modality will signal the contract to attempt to upgrade from version 1.0 to version 1.1.1. In this scenario, the contract will retrieve the package hash and the access URef from the `NamedKey` entries originally created during the 1.0 installation.
-3. `V_1_0_custom`: This modality will signal the contract to attempt to upgrade from version 1.0 to version 1.1.1. In this scenario, the calling account must provide the `NamedKey` entries under which the package hash and the access URef are saved. Additionally, this requires the passing of the runtime arguments `access_key_name` and `hash_key_name` for the access URef and package hash, respectively. In this modality, these arguments are required and must be passed in.
-
-| NamedKeyConvention        | u8  |
-| ------------------------- | --- |
-| DerivedFromCollectionName | 0   |
-| V_1_0_standard            | 1   |
-| V_1_0_custom              | 2   |
-
-#### EventsMode
-
-The `EventsMode` modality determines how the installed instance of CEP-78 will handle the recording of events that occur from interacting with the contract.
-
-The modality provides three options:
-
-1. `NoEvents`: This modality will signal the contract to not record events at all. This is the default mode.
-2. `CEP47`: This modality will signal the contract to record events using the CEP47 event schema. Further information can be found [below](#cep47-mode).
-3. `CES`: This modality will signal the contract to record events using the [Casper Event Standard](#casper-event-standard).
-
-| EventsMode | u8  |
-| ---------- | --- |
-| NoEvents   | 0   |
-| CEP47      | 1   |
-| CES        | 2   |
-
-#### Transfer Filter Hook
-
-The transfer filter modality, if enabled, specifies a contract package hash pointing to a contract that will be called when the `transfer` method is invoked on the contract. CEP-78 will call the `can_transfer`
-method on the specified callback contract, which is expected to return a value of `TransferFilterContractResult`, represented as a u8.
-
-- `TransferFilterContractResult::DenyTransfer` will block the transfer regardless of the outcome of other checks
-- `TransferFilterContractResult::ProceedTransfer` will allow the transfer to proceed if other checks also pass
-
-The transfer filter can be enabled by passing a `ARG_TRANSFER_FILTER_CONTRACT` argument to the install method, with a value of type `Option<Key>`
```

Comment:
The transfer filter hook also needs to move to the modalities.md file. Perhaps there was a merge issue?
Also, I notice that this modality does not follow the formatting of the previous ones that have a table and a list of options.

Reply:
This is bizarre. I don't remember writing this or when it showed up in here - I'm going to have to take a look at the previous commits.

I'm fairly confident this wasn't in my local, updated cep-78 repo.